docs/en/conf.md

```diff
@@ -0,0 +1,189 @@
+--tablet_heartbeat_timeout=60000
+# Configure the tablet health check interval, in milliseconds
+#--tablet_offline_check_interval=1000
+
+# The number of queues to perform high-availability tasks
+#--name_server_task_pool_size=8
+# The number of concurrent execution of high-availability tasks
+#--name_server_task_concurrency=2
+# The maximum number of concurrent execution of high-availability tasks
+#--name_server_task_max_concurrency=8
+# Check the waiting time of the task when executing the task
+#--name_server_task_wait_time=1000
+# The maximum time to execute the task, if it exceeds, it will log
+#--name_server_op_execute_timeout=7200000
+# The time interval of receiving the status of the next task
+#--get_task_status_interval=2000
+# The time interval of receiving the status of the next table
+#--get_table_status_interval=2000
+# Check the minimum difference of binlog synchronization progress, if the master-slave offset is less than this value, the task has been successfully synchronized
+#--check_binlog_sync_progress_delta=100000
+# The maximum number of tasks to save, if this value is exceeded, completed and failed ops will be deleted
+#--max_op_num=10000
+
+# Create the default number of replicas for the table
+#--replica_num=3
+# Create the default number of shards for a table
+#--partition_num=8
+# The default number of replicas for system tables
+--system_table_replica_num=2
+```
+
+## tablet configuration file conf/tablet.flags
+
+```
+# tablet.conf
+# Whether to use aliases
+#--use_name=false
+# The port number to start, if the endpoint is specified, it is not necessary to specify the port
+#--port=9527
+# Startup ip/domain name and port number
+--endpoint=127.0.0.1:9921
+# The role to start, cannot be modified
+--role=tablet
+
+# If you start the cluster version, you need to specify the address of zk and the node path of the cluster in zk
+#--zk_cluster=127.0.0.1:7181
+#--zk_root_path=/openmldb_cluster
+
+# Configure the thread pool size, it is recommended to be consistent with the number of cpu cores
+--thread_pool_size=24
+# zk session timeout, in milliseconds
+--zk_session_timeout=10000
+# Interval for checking zk status, in milliseconds
+#--zk_keep_alive_check_interval=15000
+
+# log file path
+--openmldb_log_dir=./logs
+
+# binlog conf
+# Binlog wait time when no new data is added
+#--binlog_coffee_time=1000
+# Master-slave matching offset waiting time
+#--binlog_match_logoffset_interval=1000
+# Whether to notify the follower to synchronize immediately when data is written
+--binlog_notify_on_put=true
+# The maximum size of the binlog file, in M
+--binlog_single_file_max_size=2048
+# Master-slave synchronization batch size
+#--binlog_sync_batch_size=32
+# The interval between binlog sync and disk, in milliseconds
+--binlog_sync_to_disk_interval=5000
+# If there is no new data synchronization wait time, in milliseconds
```

Comment:
The wait time when there is no new data synchronization, in milliseconds

Reply:
Thank you.